### ISSUE TITLE
Differences in OpenFAST model to design report

### ISSUE THREAD

**Issue by Bruce-Cavanagh** [IEAWindTask37/IEA-10.0-198-RWT#22]
Good morning, 

I was hoping you could enlighten me as to why there are the following differences between the IEA Wind TCP Task 37 report (specifically Table 9) and the IEA-10.0-198-RWT_Elastodyn file on this repository: 
- Overhang = 7.1m (IEA report) / 10.039m (Elastodyn input file)
- Blade mass = 47700 kg (IEA report) / 48603.4 kg (Elastodyn .sum output)
- Nacelle mass = 542600 kg (IEA report) / 545623 kg (Elastodyn input file)
- Tower mass = 628442 (IEA report) / 586474.4 kg (Elastodyn .sum output)

Thanks in advance, 
Bruce

**Comment by ptrbortolotti** [comment]
Dear Bruce,
thanks for looking into the models, your questions are good ones. Let me try to help out:
1) The IEA 10MW went through various design iterations at different institutions. The rotor design happened at DTU and the process built off of the DTU 10W. That model had 7.1 m of overhang. In parallel, a series of students at NTNU in Norway redesigned the mechanical components of the nacelle. Once these two loops were completed, NREL designed the generator, and finally DTU rerun the load analysis in HAWC2. The nacelle and generator design from NTNU/NREL increased the overhang to 10.039 m, see Figure 37 of https://www.nrel.gov/docs/fy19osti/73492.pdf. 
![Screen Shot 2021-02-02 at 6 26 58 PM](https://user-images.githubusercontent.com/20641647/106684763-51f48580-6584-11eb-9eaf-2feda513b4d0.png)
Note that in OpenFAST the overhang is measured along the tilted rotor axis (https://openfast.readthedocs.io/en/master/source/user/aerodyn/input.html?highlight=OverHang#aerodyn-driver-input-file). Conclusion: 7.1 is wrong, 10.039 is the right value to use.
2) The blade mass of 47.7 tons came from the models Because / HAWC2. I did my best to reconstruct the same identical structure in PreComp / ElastoDyn, but differences certainly exist. I'm still on the hook to generate the BeamDyn files for the blade. That will give a third value for blade mass...
3) I am not sure about this difference, which however luckily is small. It was probably the result of some slight inconsistencies during the design iterations.
4) Tower mass is more concerning and I must admit that I'm not able to reconstruct where 586 tons came from. Note that the properties in Table 15 of the report are different than the original tower for the DTU 10MW, which was 628 tons and was designed without the offshore substructure. If you are interested in the tower, you might consider using the model available in the offshore branch of the repository
https://github.com/IEAWindTask37/IEA-10.0-198-RWT/tree/offshore
My NREL colleague Roger Bergua carefully built the model for the OC6 project and things have been double checked there

I hope this helps.
Best regards,
Pietro

**Comment by Bruce-Cavanagh** [comment]
Pietro, 

Thank you taking the time to respond and for your detailed answers, it is very helpful. I am satisfied with 1-3 since the difference in blade mass and nacelle are minor. I am still interested in the tower weight, the offshore model gives a tower weight of 1033t, I understand that floater towers have to be reinforced however this is considerably greater than what is stated in the IEA Wind Task 37 report. Therefore, I will likely continue with the 586t as it is closer, so I would still be interested in the reasoning behind this number, but it is not a priority, just a curiosity from my side now. 

Thank you again, 
Bruce

**Comment by ptrbortolotti** [comment]
Hi Bruce,
my apologies for the slow response. 
The 600 tons tower is the original tower from the DTU 10MW, which I believe was the output of a simple upscaling exercise.
It is therefore likely that the tower was undersized and its mass is likely unrealistically low. During my PhD at the Technical University of Munich I actually looked into it and I found several constraints failing. You can read more about it here https://wes.copernicus.org/articles/1/71/2016/wes-1-71-2016.pdf (but watch out because in the paper I also increased the height).
The 1000 ton design should therefore be closer to reality as it came out of a design study conducted at NTNU in Norway.
I hope this helps.
Best regards,
Pietro

**Comment by RBergua** [comment]
Leaving this here for the offshore tower in case it's useful for anyone in the future:
The Norwegian University of Science and Technology (NTNU), in partnership with the NOWITECH Research Centre for Offshore Wind Technology, developed the offshore tower as well as the monopile for the DTU-10 MW RWT. The IEA-10.0-198-RWT uses the same tower definition.

From Anaya‐Lara, O., J.O. Tande, K. Uhlen, and K. Merz. 2020. Appendix in Offshore Wind Energy Technology (eds O. Anaya‐Lara, J.O. Tande, K. Uhlen, and K. Merz): 
![image](https://github.com/IEAWindTask37/IEA-10.0-198-RWT/assets/33709140/2a3983d4-eec2-4402-bca1-65f4af2d778f)

The OC6 Phase II project also used that tower design. The definition is available in Tables 1 and 2 here: https://www.nrel.gov/docs/fy21osti/79938.pdf. For reference, the corresponding eigenfrequencies for the system are available in Table 3 of the same document.

I hope that helps.
